src/Function.cpp

```diff
@@ -77,6 +77,53 @@ static long cur_func_idx;				// Index into FuncList that we are currently workin
 static bool param_first=true;			// Flag to track output of commas
 static int builtin_functions_cnt;
 
+void
+FunctionAttribute::OutputAttribute(std::ostream &out)
```

Comment:
Please add one additional argument where you'll pass additional attribute value.